views/subscribers/importExport/import.html

```diff
@@ -133,5 +133,11 @@ <h1 class="title">
 'offerClearoutText3': __('We recommend cleaning your list with Clearout because the first 500 emails are free, it’s fast and it detects up to 50% of invalid addresses. It’s our support team’s favorite.'),
 'tryClearout': _x('Try clearout.io for free', 'CTA button label'),
 'clearoutGotIt': _x('Got it, I’ll proceed to import', 'Text in a link'),
+'subscribed': __('Subscribed'),
+'unsubscribed': __('Unsubscribed'),
+'inactive': __('Inactive'),
+'dontUpdate': __('Don\’t update'),
+'newSubscribersStatus': __('Set new subscribers\' status to'),
```

Comment:
The same note about the `\'` here. We should use a typographical apostrophe because it looks better on the page and we don't need to escape it.